src/Core/src/Handlers/ScrollView/ScrollViewHandler.cs

```diff
@@ -29,7 +29,8 @@ public partial class ScrollViewHandler : IScrollViewHandler
 
 		public static CommandMapper<IScrollView, IScrollViewHandler> CommandMapper = new(ViewCommandMapper)
 		{
-			[nameof(IScrollView.RequestScrollTo)] = MapRequestScrollTo
+			[nameof(IScrollView.RequestScrollTo)] = MapRequestScrollTo,
+			[nameof(IView.InvalidateMeasure)] = MapInvalidateMeasure,
```

Comment:
can we if/def this here so that it's only windows?
```suggestion
#if WINDOWS
			[nameof(IView.InvalidateMeasure)] = MapInvalidateMeasure,
#endif			
```

Reply:
Yep, that is reasonable